py_gtfs_rt_ingestion/py_gtfs_rt_ingestion/postgres_utils.py

```diff
@@ -91,13 +173,26 @@ def insert_metadata(written_file) -> None:  # type: ignore
     process_logger = ProcessLogger("metadata_insert", filepath=filepath)
     process_logger.log_start()
 
+    metadataTable = sa.Table(
+        "metadataLog",
+        sa.MetaData(),
+        sa.Column("pk_id", sa.Integer, primary_key=True),
+        sa.Column("processed", sa.Boolean, default=sa.false()),
+        sa.Column("path", sa.String(256), nullable=False, unique=True),
+        sa.Column(
+            "created_on",
+            sa.DateTime(timezone=True),
+            server_default=sa.func.now(),
+        ),
+    )
+
     try:
-        with get_psql_conn() as connection:
-            with connection.cursor() as cursor:
-                cursor.execute(
-                    'INSERT INTO "metadataLog" (processed, path) VALUES (%s, %s)',
-                    (False, filepath),
-                )
+        with get_local_engine().connect() as connection:
+            stmt = sa.insert(metadataTable).values(
+                processed=False, path=filepath
+            )
+            connection.execute(stmt)
+            connection.commit()  # type: ignore
```

Comment:
update how we're inserting metadata. seems to work. i ran a test on aws and checked the rds after running it.